src/llvm/lib_llvm.cr

```diff
@@ -10,7 +10,7 @@ end
   {% end %}
   @[Link(ldflags: {{"`#{LibLLVM::LLVM_CONFIG} --libs --system-libs --ldflags#{" --link-static".id if flag?(:static)}#{" 2> /dev/null".id unless flag?(:win32)}`"}})]
   lib LibLLVM
-    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify}}
+    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.gsub(/git/,"")}}
```

Comment:
Could be tweaked a bit:

```suggestion
    VERSION = {{ `#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.sub(/git$/, "") }}
```

Reply:
Hm, actually there's no need for a regex:
```suggestion
    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.sub("git","")}}
```

Reply:
There is no `StringLiteral#sub`, either

Reply:
@straight-shoota It is, if you want to remove the `git` from the **end** of the string.

Reply:
That wouldn't work either, the macro `StringLiteral#gsub` requires a `RegexLiteral` first argument

Reply:
And of course, moving the `.gsub` or `.sub` outside `{{ ... }}` isn't an option as `LibLLVM::VERSION`'s initializer must be a literal

Reply:
Okay, so we leave everything as it is. Sorry for all the confusion 🙇

Reply:
IMO `$` should be added, still.